client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -282,37 +282,6 @@ const PluginDetailsCTA = ( { plugin, isPlaceholder } ) => {
 		);
 	}
 
-	if ( ! selectedSite && isLoggedIn ) {
-		// Check if there is no site selected
-		return (
-			<div className="plugin-details-cta__container">
```

Comment:
Introduce the `ManageSitesButton` component and use it inside the `PrimaryButton`